bluebinder_wait.sh

```diff
@@ -24,6 +24,12 @@ do
             chown root:root /var/lib/bluetooth/board-address
             chmod 644 /var/lib/bluetooth/board-address
             exit 0
+        elif [ "$(getprop persist.vendor.service.bdroid.bdaddr)" != "" ]; then
+            mkdir -p /var/lib/bluetooth
+            echo $(getprop persist.vendor.service.bdroid.bdaddr |sed "s/://g" | fold -w 2 | tac | paste -sd '\0' - |sed 's/.\{2\}/&:/g' |sed "s/:$//g") > /var/lib/bluetooth/board-address
```

Comment:
Is this pipe using 4 different tools just to change the byte order, e.g. `11:22:33:44:55:66` -> `66:55:44:33:22:11`?

Reply:
If that's the case, then this solution from stackoverflow.com looks more elegant to me:
```
awk -F: '{do printf "%s"(NF>1?FS:RS),$NF;while(--NF)}'
```

Reply:
@monich thanks, changed it